organization/components/timeslot/TurfCard.tsx

```
@@ -0,0 +1,92 @@
+export default function TurfCard({ turf, isSelected, onSelect }: TurfCardProps) {
+  const firstImage = turf.images.length > 0 ? turf.images[0] : '/placeholder-turf.jpg';
+
+  return (
+    <div
+      className={`relative overflow-hidden rounded-xl transition-all ${
+        isSelected 
+          ? 'ring-2 ring-blue-500 shadow-lg' 
+          : 'hover:shadow-md border border-gray-200'
+      }`}
+      onClick={onSelect}
+    >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)